src/HealthcareApis/HealthcareApis/help/Get-AzHealthcareApisService.md

```diff
@@ -0,0 +1,236 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.HealthcareApis.dll-Help.xml
+Module Name: Az.HealthcareApis
+online version: https://docs.microsoft.com/en-us/powershell/module/az.healthcareapisservice/get-azhealthcareapisservice
+schema: 2.0.0
+---
+
+# Get-AzHealthcareApisService
+
+## SYNOPSIS
+Get the metadata of a service instance.
+
+## SYNTAX
+
+### ListParameterSet (Default)
+```
+Get-AzHealthcareApisService [-ResourceGroupName <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### ServiceNameParameterSet
+```
+Get-AzHealthcareApisService -ResourceGroupName <String> -Name <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
+```
+Get-AzHealthcareApisService -ResourceId <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Gets existing healthcareApis fhir service accounts created within the specified subscription or a resource group.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-AzHealthcareApisFhirService -Name "MyService" -ResourceGroupName "MyResourceGroup"
```

Comment:
@poadhika it looks like all of your examples use `HealthcareApisFhirService` instead of `HealthcareApisService` -- this should be updated in all examples across your cmdlets